app/controllers/api/v1/targets_controller.rb

```diff
@@ -22,6 +22,16 @@ def target_params
         params.require(:target).permit(:title, :radius, :longitude, :latitude, :topic_id)
       end
 
+      def target_attibutes
```

Comment:
1. There's a typoe here (target_attibutes -> `target_attributes`)
2. What are you using this for?